packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -63,6 +63,11 @@ const propTypes = {
    * The first parameter is the event. The second parameter is the changed input value.
    */
   onChangeRaw: PropTypes.func,
+  /**
+   * A callback function to execute when a selection is made in the date picker.
+   * The first parameter is the event. The second parameter is the selected input value.
```

Comment:
nit: should this be update to indicate the returned input value is formatted or would consumers be expecting the formatted date?

Below:
```
if (this.props.onSelect) {
   this.props.onSelect(event, updatedDateTime.format());
}
```

Reply:
I agree to update the comment that the returned value is in the ISO format. The props takes an ISO format so I think it makes sense that onSelect also returned the value in ISO format similar to the onSelect event in DatePicker and the onChange event.)

Reply:
Updated here https://github.com/cerner/terra-framework/pull/585/commits/bf7770d8eb04e4701794086433d6b1f6f1609afe